src/Deposit.js

```diff
@@ -392,7 +413,10 @@ export default class Deposit {
         console.debug(
           `Monitoring Bitcoin for transaction to address ${address}...`
         )
-        return BitcoinHelpers.Transaction.findOrWaitFor(address, expectedValue)
+        return BitcoinHelpers.Transaction.findOrWaitFor(
+          address,
+          expectedValue.toNumber()
```

Comment:
This worked before because JavaScript will coerce a value into a number when needed; this lets us be more explicit, however.